FrenchFilter/sections/antiadblock.txt

```diff
@@ -17,6 +17,8 @@ capital.fr#@#.plainAd
 !
 !
 !
+! https://www.reddit.com/r/uBlockOrigin/comments/v2b6l4/asking_to_unblock_the_adblocker/
+francaisfacile.net#%#//scriptlet("prevent-addEventListener", "DOMContentLoaded", "Adblock")
```

Comment:
We already have the rules for this. FIxed https://github.com/AdguardTeam/AdguardFilters/commit/c6c13d41d6f145518b9c5fea969a5ffb6639362d